contracts/transmuter/facets/Getters.sol

```diff
@@ -156,11 +156,11 @@ contract Getters is IGetters {
 
     /// @inheritdoc IGetters
     function isWhitelistedForType(WhitelistType whitelistType, address sender) external view returns (bool) {
```

Comment:
why not removing this function totally or renaming it?

Reply:
it a bit weird like this because it doesn't check keiring for backed

Reply:
So people can still call it locally using the trick to make non view calls static